package.json

```diff
@@ -2,58 +2,51 @@
   "name": "@playkit-js/playkit-js-providers",
   "version": "2.39.4",
   "description": "",
+  "exports": {
```

Comment:
does this change require you to also make changes in the imports in packages that use one of the dist exports, like kava plugin which uses analytics service ?